torch/fx/graph.py

```diff
@@ -648,8 +648,31 @@ def emit_node(node: Node):
                     "val",
                     node.meta.get("tensor_meta", node.meta.get("example_value", None)),
                 )
+                # Tensor subclass printing support
+                if (
+                    issubclass(meta_val.__class__, torch.Tensor)
+                    and meta_val.__class__ != torch.Tensor
+                    and isinstance(meta_val, torch.Tensor)
+                    # Fake tensors cause tests to wobble and the benefits to end
+                    # users are unclear so let's not custom print them for now.
+                    and not isinstance(meta_val, torch._subclasses.FakeTensor)
+                ):
+                    # Get the basic tensor representation
+                    stride_annotation = (
```

Comment:
if we're just going to print the subclass name and not the other medata, we can probably simplify this. stride/device/shape are all generated from the other codepath too?